readthedocs/gold/models.py

```diff
@@ -61,3 +66,9 @@ def num_supported_projects(self):
         dollars = int(self.level.split('-')[-1])
         num_projects = int(math.floor(dollars // DOLLARS_PER_PROJECT))
         return num_projects
+
+    def can_sponsor_projects(self):
```

Comment:
I'm going to make a small change here such that if we add a gold sponsorship in the Django admin that this should return True